packages/svelte/src/escaping.js

```diff
@@ -0,0 +1,26 @@
+const ATTR_REGEX = /["<]/g;
```

Comment:
Don't we want to be escaping `&` in attributes?

Something like `<div title="&amp;lt;">blah</div>` looks like it renders incorrectly currently.

Reply:
So, it looks like that sort of code already was handled incorrectly in Svelte 4 in server mode but it was handled correctly in client mode.

In Svelte 5, it looks like it's handled incorrectly in both server and client modes.

Reply:
Not sure what you mean, it seems render fine for me if I do `<div title="&<">blah</div>`

Reply:
`<div title="&amp;lt;">blah</div>`, not `<div title="&<">blah</div>`. (Did you read an email or some notification version of my comment that was itself not escaping special characters properly?)

Take a look at this Svelte 5 REPL - https://svelte-5-preview.vercel.app/#H4sIAAAAAAAAChWKSw5AMBCGryL_wkpiX4_EObAoHTHJlEaHRMTd1fJ7PFhYKML0DzbrCQZdCCigd_ghXiRKieN-HvNvasdXpqxCzYDc-lCJVgPaSexalym26fa744XJwehx0ju-H3x_PN1oAAAA - This has the incorrect behavior.

Reply:
Ah yeah my email client escaped them lol. Thanks!